lib/nfg_ui/bootstrap/components/modal.rb

```diff
@@ -39,6 +43,10 @@ def render
           end
         end
 
+        def render_in_body
```

Comment:
@kylejmorin just noting that I've piggybacked off of the render_in_body technique we used on the tile so that modal footers are now usable (and thus, closable from the footer). Neat, eh?

Reply:
@jonathanroehm great! That will be helpful later on. I can see a scenario where we have a modal with some language and then action button(s) that requires the user's action without the ability to close the modal.